frontend/src/config.ts

```diff
@@ -188,6 +194,8 @@ const setConfig = ({ fetch, urlOverride, jwt, isServerless, featureFlags, ...arg
     transformsClient: transformClient,
     rpcnSecretsClient: secretGrpcClient,
     clusterStatusClient: clusterStatusGrpcClient,
+    knowledgebaseClient: knowledgebaseGrpcClient,
+    userClient: userGrpcClient,
```

Comment:
What do we use `userClient` for?

Reply:
i added it for the "auto create users", but had to stop implementing it, it was a bit too tricky.
i can remove the client - i did keep it because it's reasonable to have this client available